README.md

```diff
@@ -1,11 +1,11 @@
 [![Build Status](https://travis-ci.org/usyd-blockchain/vandal.svg?branch=master)](https://travis-ci.org/usyd-blockchain/vandal)
 
-# Vandal: A static program analysis framework for EVM bytecode
+# Vandal
 
-This repository contains an Ethereum VM bytecode analysis framework called Vandal.
+Vandal is an Ethereum VM static program analysis framework.
 It takes EVM bytecode or disassembly as input, and outputs an equivalent
-intermediate representation, including the program's control flow
-graph. This intermediate representation removes all stack operations and
+decompiled intermediate representation, including the program's control flow
```

Comment:
Hmm. This bit sounds awkward to me. The IR isn't decompiled, so "decompiled IR" sounds weird. How about something like:

> It takes EVM bytecode or disassembly as input, and performs decompilation to an equivalent intermediate representation,

Reply:
How about:

> It takes EVM bytecode or disassembly as input, and decompiles it to an equivalent intermediate representation, ...

Reply:
Sounds good